eng/pipelines/steps/PublishOpenDebugAD7.yml

```diff
@@ -6,15 +6,17 @@ parameters:
 steps:
 - script: |
     dotnet publish $(Build.SourcesDirectory)\src\OpenDebugAD7\OpenDebugAD7.csproj -c ${{ parameters.Configuration }} -r ${{ parameters.RuntimeID }} --self-contained -o $(Build.StagingDirectory)\${{ parameters.RuntimeID }}\debugAdapters\bin
+    copy ${{ parameters.SignedBinariesFolder }}\Release\vscode\OpenDebugAD7.dll "$(Build.StagingDirectory)\${{ parameters.RuntimeID }}\debugAdapters\bin\.
```

Comment:
Do you need this? I would have thought the publishing process would do this automatically.

Reply:
Yes. The `dotnet publish` overwrites the signed `OpenDebugAD7.dll` that was built by msbuild and drops a new copy.